### PR TITLE
Add security advisories video.

### PR DESCRIPTION
**Pull request by amouat**
## Type of change
Add page for security advisories video

### Why are we making this change?

Video had no home!

### How should this PR be tested? 
Please check links and text for typos

### CODE REVIEW

**Review by SharpRake — APPROVED** [review]
This is great! just a couple minor tweaks

File: content/chainguard/chainguard-images/videos/security_advisories.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+affecting Chainguard images"
+type: "article"
+date: 2024-01-18T01:21:01+00:00
+lastmod: 2024-01-18T15:21:01+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 10
```
[the remark this answers is not in the document]
Comment (amouat) [reply content/chainguard/chainguard-images/videos/security_advisories.md:15]: Ha, I didn't even know what the weight did 🤦 

Yeah, I'll just change this one.

**Comment by amouat** [comment]
@SharpRake Thanks for the transcript changes as well. YouTube is annoying; I wish the lines were twice as long. I might update the script I have to merge lines for new videos.

File: content/chainguard/chainguard-images/videos/security_advisories.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amouat) [reply content/chainguard/chainguard-images/videos/security_advisories.md]: Done, after a false start (apparently heavy things fall to the bottom, who knew?)

**Pull request merged**